index.css

```diff
@@ -0,0 +1,108 @@
+html body {
+  margin: 0;
+  padding: 0;
+}
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.background-1 {
+  position: absolute;
+  right: 0;
+  height: 256px;
+  top: 430px;
+}
```

Comment:
- [ ] [Optional] Kindly be noticed that you can minimize the height of the background image to make it not go over the div.
```css
.background-1 {
  position: absolute;
  right: 0;
  height: 210px;
  top: 430px;
}
```